src/supervisor.adoc

```diff
@@ -362,6 +362,13 @@ An interrupt _i_ will trap to S-mode if both of the following are true:
 `sstatus` register is set, or the current privilege mode has less
 privilege than S-mode; and (b) bit _i_ is set in both `sip` and `sie`.
 
+NOTE
+====
+Trapping to S-mode only occurs if bit _i_ of `mideleg` is set. Otherwise,
+bit _i_ of `sip` and `sie` are read-only zero so condition (b) cannot be
+be satisfied.
```

Comment:
Not always true. 
1. When mideleg[i] is 1, sip[i] and sie[i] are aliases of mip[i] and mie[i] respectively.
2. When mideleg[i] is 0 and mvien[i] is 1, sip[i] is an alias of mvip[i] and sie[i] is writable. 
3. When mideleg[i] and mvien[i] are both 0, sip[i] and sie[i] are both read-only zero.

Reply:
This note however breaks the layering of the ISA. S level ISA should be orthogonal to how sip gets set. Otherwise every future  addition like mvien would need to be updated/repeated in multiple places